spec/rails_helper.rb

```diff
@@ -15,7 +15,7 @@
 
     # The intention of this value is that it should never go down after a PR
     # It is a (very) naive attempt to prevent untested code entering the codebase
-    minimum_coverage 98.82
+    minimum_coverage 98.75
```

Comment:
@starswan is it absolutely necessary to reduce test coverage? the natural preference is to maintain/improve coverage

Reply:
Yes - the massive reduction in code lines results in the coverage percentage dropping e.g. 2000/2500 is 80%, but delete 1000 covered lines from that and you have 1000/1500 66%